hoodie-spark/src/main/java/com/uber/hoodie/DataSourceUtils.java

```diff
@@ -142,4 +143,19 @@ public static HoodieRecord createHoodieRecord(GenericRecord gr, Comparable order
     HoodieRecordPayload payload = DataSourceUtils.createPayload(payloadClass, gr, orderingVal);
     return new HoodieRecord<>(hKey, payload);
   }
+
+  @SuppressWarnings("unchecked")
```

Comment:
@leletan I reused your method here.. if I land first, then you can just add a new overload that takes parameters and call this..

Reply:
@vinothchandar Ack. Thanks for letting me know